src/sidebar/util/time.js

```diff
@@ -195,48 +198,49 @@ function nextFuzzyUpdate(date, now) {
  * update frequency depends on the age of a timestamp.
  *
  * @param {String} date - An ISO 8601 date string timestamp to format.
- * @param {Date} callback - A callback function to call when the timestamp changes.
+ * @param {Function} callback - A callback function to call when the timestamp changes.
  * @return {Function} A function that cancels the automatic refresh.
  */
 function decayingInterval(date, callback) {
   let timer;
   const timeStamp = date ? new Date(date) : null;
-  const update = function() {
+
+  const update = () => {
     const fuzzyUpdate = nextFuzzyUpdate(timeStamp, new Date());
     if (fuzzyUpdate === null) {
       return;
     }
-    const nextUpdate = 1000 * fuzzyUpdate + 500;
-    timer = setTimeout(function() {
+    const nextUpdate = fuzzyUpdate + 500;
+    timer = setTimeout(() => {
       callback(date);
       update();
     }, nextUpdate);
   };
+
   update();
 
-  return function() {
-    clearTimeout(timer);
-  };
+  return () => clearTimeout(timer);
 }
 
 /**
  * Formats a date as a string relative to the current date.
  *
  * @param {Date} date - The date to consider as the timestamp to format.
  * @param {Date} now - The date to consider as the current time.
- * @param {Object} Intl - JS internationalization API implementation.
+ * @param {Object} Intl - JS internationalization API implementation; this
+ *                      param is present for dependency injection during test.
  * @return {string} A 'fuzzy' string describing the relative age of the date.
  */
 function toFuzzyString(date, now, Intl) {
   if (!date) {
     return '';
   }
-  return getBreakpoint(date, now).format(date, now, Intl);
+  return getBreakpoint(date, now).formatFn(date, now, Intl);
 }
 
 module.exports = {
-  clearFormatters: clearFormatters,
+  clearFormatters: clearFormatters, // For testing
```

Comment:
This and `nextFuzzyUpdate` are never used in app code. They're here as affordances for tests. It gives me the wiggles a little bit but I don't have a better idea right now.

Reply:
Looks like nextFuzzyUpdate is used in decayingInterval, no?

Reply:
Sorry, I should have said "never used externally in app code." i.e. doesn't need to be exported for use in the app; it's only there for testing.